docs/src/main/asciidoc/getting-started-guide.adoc

```diff
@@ -106,20 +106,18 @@ In addition, you can see the `quarkus-maven-plugin` responsible of the packaging
 </build>
 ----
 
-If we focus on the dependencies section, you can see 2 extensions allowing the development of REST applications:
+If we focus on the dependencies section, you can see one of the 2 extensions allowing the development of REST applications:
 
 [source,xml]
 ----
     <dependency>
         <groupId>io.quarkus</groupId>
         <artifactId>quarkus-resteasy</artifactId>
     </dependency>
-    <dependency>
-        <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-arc</artifactId>
-    </dependency>
 ----
 
+The other one is the `quarkus-arc` brought by the `quarkus-bom` POM dependency imported in the project.
+
```

Comment:
I think it would be better to introduce ArC later, when we are introducing the service. Typically in this paragraph: https://github.com/quarkusio/quarkus/pull/1651/files#diff-675231b187331ca14a503802fab8a51bR198 .

It's not brought by the bom but it is a dependency of `quarkus-resteasy`.

Reply:
@gsmet is it better now?